From 724b4c1819e833b22f7546c9768e1034996fa337 Mon Sep 17 00:00:00 2001
From: Nathan Owens <ndowens@artixlinux.org>
Date: Wed, 30 Dec 2020 16:48:44 -0600
Subject: [PATCH] fix cross build

AC_TRY_RUN will fail with cross, 
use AC_COMPILE_IFELSE instead
---
diff --git configure.ac configure.ac
index 60fc968..d6e5eee 100644
--- configure.ac
+++ configure.ac
@@ -159,7 +159,7 @@ LIBS=$RT_LIBS
 
 AC_MSG_CHECKING(for CLOCK_MONOTONIC)
 
-AC_TRY_RUN([
+AC_COMPILE_IFELSE([
   #include <time.h>
   #include <errno.h>
 main()
-- 
2.30.0

